licensing/build.gradle

```diff
@@ -31,4 +35,19 @@ dependencies {
     compile project(':core')
     testCompile "junit:junit:${versions.junit}"
     testCompile project(':integration-testing')
-}
\ No newline at end of file
+}
+
+processResources {
+  String cratePublicKey = System.getProperty("crate.public.key")
+  if (cratePublicKey != null) {
+    println "Processing provided crate public key ${cratePublicKey}"
+    if (Files.exists(Paths.get(cratePublicKey)) == false) {
+        throw new IllegalArgumentException('crate.public.key at specified path [' + cratePublicKey + '] does not exist')
+      }
+      from(cratePublicKey) {
+        rename { String filename -> 'public.key' }
```

Comment:
not sure how this works - how are we sure that this will be copied to the main/resources folder?